packages/data/src/world/oot/jabu_jabu.yml

```diff
@@ -7,10 +7,10 @@
   dungeon: JJ
   exits:
     "Jabu-Jabu": "true"
-    "Jabu-Jabu Pre-Boss": "event(BIG_OCTO) || (has_hover_boots && trick(OOT_JABU_BOSS_HOVER)) || climb_anywhere || (glitch_megaflip && trick(OOT_JJB_BOXLESS) && can_jump_slash)"
+    "Jabu-Jabu Pre-Boss": "event(BIG_OCTO) || (has_hover_boots && (trick(OOT_JABU_BOSS_HOVER) || can_play_elegy)) || climb_anywhere || (glitch_megaflip && trick(OOT_JJB_BOXLESS) && can_jump_slash)"
```

Comment:
Should we add can_play_elegy to the last part as an alternative to the JJB_BOXLESS trick?

Reply:
oh, right. probably should.